docker-compose.yml

```diff
@@ -37,6 +42,20 @@ services:
       - network.host=0.0.0.0
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - ELASTICSEARCH_URL=https://elasticsearch:9200
```

Comment:
I think it should be:
`ELASTICSEARCH_PROTOCOL="https"`
`ELASTICSEARCH_IP="elasticsearch"`
`ELASTICSEARCH_PORT="9200"`

Then, you can build a variable (not as input):
`ELASTICSEARCH_URL="$ELASTICSEARCH_PROTOCOL://$ELASTICSEARCH_IP"`

Reply:
Changed